templates/support_channels.html

```diff
@@ -6,295 +6,46 @@
 {% brand_institution_name as institution %}
 
 <section class="frontpage frontpage-section" aria-labelledby="students-title">
-	<h2 id="students-title">
-		{% translate "STUDENTS" %}
-	</h2>
-	<p>
-		{% blocktranslate trimmed %}
-		CONTACT_COURSE_STAFF_FOR_ASSISTANCE_ON_COURSE_MATTERS
-		{% endblocktranslate %}
-	</p>
 	<div class="cards">
-		<div class="card frontpage">
+		<div class="widecard frontpage">
 			<div class="card-body">
-				<h3>{% translate "STUDIES" %}</h3>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_STUDIES_LINK' %}">
-						{% translate "STUDIES" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://courses.aalto.fi">
-						{% translate "COURSE_CATALOGUE" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_STUDENT_SERVICES_LINK' %}">
-						{% translate "STUDENT_SERVICES" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_SUPPORT_FOR_STUDYING_LINK' %}">
-						{% translate "SUPPORT_FOR_STUDYING" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_STUDY_SKILLS_LINK' %}">
-						{% translate "STUDY_SKILLS" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_STARTING_POINT_OF_WELLBEING_LINK' %}">
-						{% translate "STARTING_POINT_OF_WELLBEING" %}
-					</a>
-				</p>
-			</div>
-		</div>
-
-		<div class="card frontpage">
-			<div class="card-body">
-				<h3>{% translate "IT_SUPPORT" %}</h3>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_IT_HELP_LINK' %}">
-						{% translate "IT_HELP" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://vdi.aalto.fi/">
-						{% translate "VIRTUAL_DESKTOPS" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://computers.aalto.fi/">
-						{% translate "AVAILABLE_CLASSROOM_COMPUTERS" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="https://download.aalto.fi">
-						{% translate "DOWNLOAD_LICENSED_SOFTWARE" %}
-					</a>
-				</p>
-				<p class="card-text">
-					<a target="_blank" rel="noopener" href="{% translate 'SUPPORT_AALTO_SPACE_APP_LINK' %}">
-						{% translate "AALTO_SPACE_APP" %}
-					</a>
-				</p>
-			</div>
-		</div>
-	</div>
-</section>
-<hr>
-<section class="frontpage frontpage-section" aria-labelledby="teachers-title">
-	<h2 id="teachers-title">
-		{% translate "TEACHERS" %}
-	</h2>
-	<div class="cards">
-		<div class="card frontpage">
-			<div class="card-body">
-				<h3>{% translate "DOCUMENTATION" %}</h3>
+				<h3>{% translate "TEACHERS" %}</h3>
```

Comment:
Accessibility/WebCAG requires that headings are used in the correct order and nested accordinly: h1, h2, h3. Are you now skipping the h2 level when you removed a lot of old headings? The base template (outside this file) introduces the h1 heading at the top of the page.